File: app/src/main/java/it/liceoarzignano/bold/ui/recyclerview/HeaderViewHolder.java
package it.liceoarzignano.bold.ui.recyclerview;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import it.liceoarzignano.bold.R;

public class HeaderViewHolder extends RecyclerView.ViewHolder {

    private final TextView mTitle;

    public HeaderViewHolder(View view) {
        super(view);
        mTitle = (TextView) view.findViewById(R.id.subheader_title);
    }

    public void setTitle(String title) {
        mTitle.setText(title);
    }
}

File: app/src/main/kotlin/it/liceoarzignano/bold/ui/recyclerview/HeaderViewHolder.kt
package it.liceoarzignano.bold.ui.recyclerview

import android.view.View
import android.widget.TextView
import it.liceoarzignano.bold.R

class HeaderViewHolder(view: View) : androidx.recyclerview.widget.RecyclerView.ViewHolder(view) {
    private val mTitle: TextView = view.findViewById(R.id.subheader_title)

    fun setTitle(title: String) {
        mTitle.text = title
    }
}
